bi-connectors/PowerBIConnector/power_bi_support.md

```diff
@@ -4,8 +4,8 @@
 * Microsoft Power BI Desktop
 * [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
 * [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
-* [OpenSearch.mez](../../../bi-connectors/PowerBIConnector/bin/Release)
-* Optional: [sqlodbc_import.pbids](../../../bi-connectors/PowerBIConnector/PBIDSExamples) to help with repeated connections to the same server 
+* [OpenSearchProject.mez](OpenSearchProject.mez) or [AmazonOpenSearchService.mez](AmazonOpenSearchService.mez)
```

Comment:
In the SETUP section below, it references the OpenSearch.mez file that should be updated (and make sure the folder mentioned is correct).

Reply:
Thanks, fixed in 76313580.